src/styles/index.ts

```diff
@@ -1512,6 +1512,11 @@ const styles = (theme: ThemeColors) =>
             height: '100%',
         },
 
+        canvasContainer: {
+            borderWidth: 1,
```

Comment:
What does this border do exactly? Can you point it out to me in a screenshot? Thanks!

Reply:
Ah interesting, that works for me. Just curious though, how does it fix the line bug?

Reply:
Neat! Thanks for explaining :)

Reply:
Not A Blocker; but maybe worth adding a comment here in the code summarizing this